src/dos/cdrom_image.cpp

```diff
@@ -16,6 +16,11 @@
  *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
  */
 
+// #define DEBUG 1
+#ifdef DEBUG
+#include <time.h>
+#include <chrono>
```

Comment:
I know rest of DOSBox code does not use correct C++-style includes, but maybe let's put `#include <ctime>` here?

`chrono` is part of C++11 :( Let's use this opportunity to ask Qbix *again* if he still does not want C++11.

Reply:
Yes, there isn't much value added here. Think I'd rather use some very compelling improvements that are either far fewer lines or significantly more readable.